files/qemu_check.py

```diff
@@ -35,8 +37,22 @@ def command(cmd, timeout_sec):
     proc.kill()
     return work
 
+if arch == 'amd64':
+    vmimage = "/tmp/gentoo.qcow2"
+    cmd_qemu = 'qemu-system-x86_64 -m 128M -kernel ' \
+        'linux-amd64-build/arch/x86/boot/bzImage' \
+        ' -nographic -serial mon:stdio -hda ' + vmimage + \
+        ' -append "root=/dev/sda1 console=ttyS0,115200n8 console=tty0"'
+elif arch == 'arm':
+    vmimage = "/tmp/stage3-armv7a_hardfp-latest.qcow2"
+    cmd_qemu = 'qemu-system-arm -M vexpress-a9 -smp 2 -m 1G -kernel ' \
+        'linux-arm-build/arch/arm/boot/zImage' \
+        ' -dtb linux-arm-build/arch/arm/boot/dts/vexpress-v2p-ca9.dtb' \
+        ' -sd ' + vmimage + ' -nographic -append "console=ttyAMA0,115200' \
+        ' root=/dev/mmcblk0 rootwait"'
```

Comment:
eventually it would be nice to read this from a config file?

Reply:
I agree, ideally, it would be nice to have arch/<architecture>/config.py or something like this and import the config as a python package. All architecture specific stuffs (cross compilation, qemu instance, etc) could be put into this package. I will make a proposal for this